docs/source/system_reference_guide/custom-environments.ipynb

```diff
@@ -149,7 +149,19 @@
     ".. note::\n",
     "Example config file for installing maap-py via pip [here](./example_conda_configuration_files/env-with-maap-py.yml)\n",
     "\n",
-    "MAAP users typically use the python `maap-py`. It's pre-installed in all workspaces, in the default conda environment.  Any custom environment should specify it, otherwise it is not going to be accessible from that environment. However, `maap-py` is not packaged in a public package repository, like `PyPI` or `conda-forge`. It is possible to install it directly from its github repository with `pip` though. See the configuration example linked. You can note that in the example, `maap-py` is 'versioned' using a commit hash (at the end of the github URL). "
+    "MAAP users typically use the python `maap-py`. It's pre-installed in all workspaces, in the default conda environment.  Any custom environment should specify it, otherwise it is not going to be accessible from that environment. However, `maap-py` is not packaged in a public package repository, like `PyPI` or `conda-forge`. It is possible to install it directly from its Github repository with `pip` though. See the configuration example linked. \n",
+    "\n",
+    "You can note that in the example, `maap-py` is 'versioned' using a commit hash (at the end of the Github URL).  You can find a commit code on the commits page in the maap-py Github repo (https://github.com/MAAP-Project/maap-py/commits/master/). Use the `Copy full SHA for...` button and paste it at the end of the line you see in the example.\n",
+    "\n",
+    "![Copy full SHA hash for commit](../_static/github_copy_full_commit_sha.png)\n",
+    "\n",
+    "For example, if you copy the full SHA for the first commit in this screenshot, you will have `ea8c575f5480cee2650afbfd4494aa174b7fb396` in your clipboard. Paste this into the pip command as below:\n",
```

Comment:
Should we instead be encouraging users to use certain versions of maap-py? Like v3.1.5?

Reply:
Yes I don't think they should ever over-ride the maap.py version because we don't guarantee that it is stable in commits between versions (it should be in theory, but rigorous testing hasn't been done on it yet between versions)

Reply:
@emileten how do you feel about removing this final section of the doc? I am actually not sure how it fits in. maap-py should be in any workspace by default. Would this be for doing local development (outside a workspace)?

Reply:
@rtapella @grallewellyn 

> maap-py should be in any workspace by default

Is it going to be also available in a _custom_ environment created from scratch by a user though ? 

If the answer is yes, I'll remove this section.

Reply:
@emileten maap-py will _not_ be available in a custom environment created from scratch by the user, good point

I think we should keep this section, but tell users how to install a specific version of maap-py instead of using in a commit hash
In our v4 release coming out in a couple months, maap-py should be able to be installed via pip

Reply:
we could keep this section but point people at the commit-hash link on the Releases page instead of the general commit history:
<img width="1064" alt="Screenshot 2024-04-18 at 8 43 56 AM" src="https://github.com/MAAP-Project/maap-documentation/assets/1156193/6ce667e2-cc16-4f57-a861-81582ad5ea15">

Reply:
does pip work with that "3.1.5" tag link?

Reply:
Yes, ie `pip install git+https://github.com/MAAP-Project/stac_ipyleaflet.git@0.3.6#egg-info=stac_ipyleaflet`
I agree we should link to the releases page

Reply:
@rtapella @grallewellyn I addressed this, please have a look at the last changes to this section. Thanks !

Reply:
@emileten I am not seeing the `git_tags_maap_py.png` file
Also can we link the releases page: https://github.com/MAAP-Project/maap-py/releases